instrumentation/google-http-client-1.19/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/googlehttpclient/GoogleHttpClientInstrumentationModule.java

```diff
@@ -138,15 +125,12 @@ public static void methodEnter(
         return;
       }
 
-      context = tracer().startSpan(parentContext, request, request.getHeaders());
+      context = tracer().startSpan(parentContext, request);
+      scope = context.makeCurrent();
 
       // propagating the context manually here so this instrumentation will work with and without
       // the executors instrumentation
```

Comment:
I find this comment suspicious. Why of all places here to want to work without executors instrumentations?

Reply:
I agree this comment is suspicious, so I removed it 😁.

I think we always propagate the span out-of-band to the place where it is ended. (this was actually something nice about the `Operation` concept, b/c you had to pass the Operation object out-of-band to the place where it was ended, orthogonal to in-band Context propagation)